app-admin/chezmoi/chezmoi-2.58.0.ebuild

```
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+DESCRIPTION="Manage your dotfiles across multiple diverse machines, securely."
+HOMEPAGE="https://www.chezmoi.io https://github.com/twpayne/chezmoi"
+SRC_URI="https://github.com/twpayne/chezmoi/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/tsln1998/gentoo-deps/releases/download/${P}/${P}-vendor.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 arm arm64 x86"
```

NonsolvableDepsInDev 的报错可以用 unstable keywords 来解决。

实际上 overlay 中一般不会用 stable keywords，不过 gentoo-zh 这边没有禁止。

```suggestion
KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

---

感谢提示, 已经改了 unstable